### PR TITLE
add: Mongo Express

### PR DESCRIPTION
**Pull request by berksmbl**

### CODE REVIEW

File: templates/mongo-express/index.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { Output, randomPassword, Services } from "~templates-utils";
+import { Input } from "./meta";
+
+export function generate(input: Input): Output {
+  const services: Services = [];
+  const databasePassword = randomPassword();
+
+  const appEnv = [
+    `ME_CONFIG_MONGODB_URL=mongodb://mongo:${databasePassword}@$(PROJECT_NAME)_${input.databaseServiceName}:27017`,
```
Comment (deiucanta) [inline templates/mongo-express/index.ts:9]: if you don't add this, will it ask for the connection url in the UI?

File: templates/mongo-express/index.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      projectName: input.projectName,
+      serviceName: input.appServiceName,
+      source: { type: "image", image: input.appServiceImage },
+      domains: [{ host: "$(EASYPANEL_DOMAIN)", port: 8081 }],
+      deploy: { command: "sleep 60; /docker-entrypoint.sh" },
+      env: appEnv.join("\n"),
+    },
+  });
+
+  services.push({
```
Comment (deiucanta) [inline templates/mongo-express/index.ts:32]: I would not create a mongo service along with mongo engine (similar to phpmyadmin) - if should only be the app service

**Comment by berksmbl** [comment]
@deiucanta I removed the local database.

**Review by ivanciucdragos1 — APPROVED** [review]
LGTM

File: templates/mongo-express/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  - name: Berk Sümbül
+    url: https://berksmbl.com
+schema:
+  type: object
+  required:
+    - projectName
+    - appMongoUrl
+    - appServiceName
+    - appServiceImage
+    - databaseServiceName
```
[the remark this answers is not in the document]
Comment (berksmbl) [reply templates/mongo-express/meta.yaml]: @deiucanta resolved

**Pull request merged**